### PR TITLE
ENT-4981 Add buildnumber to coreos packages

### PR DESCRIPTION
**Pull request by Lex-2008**

### CODE REVIEW

**Comment by Lex-2008** [comment]
@cf-bottom would you be so nice to build it for me?

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/4072/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/4072/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-4072/

**Comment by Lex-2008** [comment]
filenames on current master (from http://buildcache.cloud.cfengine.com/packages/testing-pr/jenkins-master-nightly-pipeline-226/PACKAGES_x86_64_linux_redhat_7/):
```
￼ cfengine-nova-3.15.0a.cf525f33b-5392.x86_64.rpm           2019-11-18 03:22  4.4M  
￼ cfengine-nova-3.15.0a.cf525f33b.x86_64.fs-img.pkg.tar.gz  2019-11-18 03:22  5.4M  
￼ cfengine-nova-3.15.0a.cf525f33b.x86_64.pkg.tar.gz         2019-11-18 03:22  5.4M  
```
filenames on this build (from http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-4072/PACKAGES_x86_64_linux_redhat_7/):
```
￼ cfengine-nova-3.15.0a.c9e02883c-5205.x86_64.rpm               2019-11-04 18:33  4.4M  
￼ cfengine-nova-3.15.0a.c9e02883c.5205.x86_64.fs-img.pkg.tar.gz 2019-11-04 18:33  5.4M  
￼ cfengine-nova-3.15.0a.c9e02883c.5205.x86_64.pkg.tar.gz        2019-11-04 18:33  5.3M  
```
Looks good to be merged!

File: build-scripts/package [hunk 1]
```diff
@@ -163,7 +163,7 @@ case "$PACKAGING" in
         "$P/$PKG.bff.sh" "$RPM_VERSION.$safe_prefix$RPM_RELEASE" "$BASEDIR" "$PREFIX"
     else
         # Create TAR package
-        TARBALL="$BASEDIR/$PKG/RPMS/$PKG-$VERSION$safe_prefix.$ARCH.pkg.tar.gz"
+        TARBALL="$BASEDIR/$PKG/RPMS/$PKG-$VERSION.$safe_prefix$RPM_RELEASE.$ARCH.pkg.tar.gz"
```
Comment (vpodzime) [inline build-scripts/package]: Any reason why it's not `-$RPM_RELEASE` just like in the RPMs?

Comment (Lex-2008) [reply build-scripts/package]: no reason. Will change!

Comment (vpodzime) [reply build-scripts/package]: I changed it.

**Pull request merged**